### ISSUE TITLE
Can't compile crate with gloo-events

### ISSUE THREAD

**Issue by kellpossible** [rustwasm/gloo#120]
### Describe the Bug

With a recent version of the following dependencies, `gloo-events` stopped compiling:

```toml
[dependencies]
gloo-events = "0.1"
wasm-bindgen = "0.2"
web-sys = "0.3"
```

```
Checking gloo-events v0.1.1
error[E0308]: mismatched types
   --> /home/luke/.cargo/registry/src/github.com-1ecc6299db9ec823/gloo-events-0.1.1/src/lib.rs:537:9
    |
534 |     pub fn forget(mut self) {
    |                             - expected `()` because of default return type
...
537 |         self.callback.take().unwrap_throw().forget()
    |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^- help: try adding a semicolon: `;`
    |         |
    |         expected `()`, found struct `wasm_bindgen::JsValue`

error: aborting due to previous error

For more information about this error, try `rustc --explain E0308`.
error: could not compile `gloo-events`.
```

### Steps to Reproduce

Create a new crate with the following dependencies:

```toml
[dependencies]
gloo-events = "0.1"
wasm-bindgen = "0.2"
web-sys = "0.3"
```

run `cargo check`

**Comment by kellpossible** [comment]
It seems likely this break is due to the following change in `wasm-bindgen` https://github.com/rustwasm/wasm-bindgen/pull/2248 and released in [0.2.66](https://github.com/rustwasm/wasm-bindgen/blob/master/CHANGELOG.md#0266)

**Comment by kellpossible** [comment]
Fixed in `wasm-bindgen` [0.2.77](https://github.com/rustwasm/wasm-bindgen/blob/master/CHANGELOG.md#0267).
